cmd/aperturectl/cmd/utils/controller.go

```diff
@@ -9,8 +9,11 @@ import (
 	"io"
 	"net/http"
 	"net/url"
+	"os"
+	"path/filepath"
 	"strings"
 
+	toml "github.com/pelletier/go-toml"
```

Comment:
Consider using https://github.com/BurntSushi/toml if possible – I see we already have it in go.mod and it has more starts on gh.